### PR TITLE
Create Installation Guide for AWS using EKS

### PR DESCRIPTION
**Pull request by summerswallow-whi**

### CODE REVIEW

**Comment by willingc** [comment]
Great! Thanks @summerswallow-whi. I'm teaching some students this summer but am looking forward to trying this out.

File: doc/source/amazon/step-zero-aws.rst [hunk 1]
```diff
@@ -197,7 +197,7 @@ Instead of performing step 13 above. Create the following ``storageclass.yml`` f
 
 The main difference is the addition of the line `encrypted: "true"` and make note that `true` is in double quotes.
 
-Next run these commands::
```
Comment (betatim) [inline doc/source/amazon/step-zero-aws.rst]: Isn't the :: part of the restructured text format?

**Comment by betatim** [comment]
How do we decide to merge this? If someone other than the author has followed this guide maybe they can post a thumbs up here.

My only question before merging: will persistent user storage work with this setup? No matter what the answer is I think mentioning at the beginning of the guide which features this way of setting up k8s gives you/doesn't give you would be good. It seems to be the most frequently asked question from people setting up new k8s clusters.

File: doc/source/amazon/step-zero-aws.rst (hunk 1 above)
Comment (summerswallow-whi) [reply doc/source/amazon/step-zero-aws.rst]: I'm new a restructured text so a lot of what I've done is best guess and can have formatting issues for sure.

**Comment by summerswallow-whi** [comment]
This will fire up an EBS for the single user notebook just as the default KOps and Heptio instructions have in the past. If you delete the notebook pod for a given user it will go away, but it will stay around after the user has logged out, same as the other creation methods.

**Comment by betatim** [comment]
There are quite a few syntax errors in this as it is. Unfortunately building the docs to highlight them so they are easy to find isn't part of our continuous integration yet. Have you tried building the docs to check what they look like locally? If not I recommend you do that and use the output to fix up the errors.

To do this create a new conda (or virtual) environment with the packages specified [here](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/doc/environment.yml) and then run `make` from the `doc/` directory.

**Comment by summerswallow-whi** [comment]
Let me see if I can run this. Good to know how to build.

**Comment by summerswallow-whi** [comment]
I'm still a little new at the fork/pull request flow, so I accidentally merged some formatting fixes to my previous contribution to the aws-kops documentation.

It's formatting and the correction of one of the commands.

**Comment by willingc** [comment]
@summerswallow-whi I'll take a look at this tomorrow (Thursday). I will probably merge as is and then do the restructured text syntax edits and then create a new PR with those (requesting review from you and @betatim).

**Comment by willingc** [comment]
Thanks @summerswallow-whi for the contribution. I'm working on a PR to clean up any syntax issues with rst and add this to table of contents. Will ping you and @betatim when done.

**Comment by consideRatio** [comment]
@summerswallow-whi :heart:

**Pull request merged**